### ISSUE TITLE
Send access card number to attendance API if available

### ISSUE THREAD

**Issue by kberzinch** [RoboJackets/apiary-mobile#68]
If the app reads a plastic BuzzCard with both a GTID and access number encoded, provide both values to the attendance API

Server-side support is part of https://github.com/RoboJackets/apiary/pull/4594

**Comment by evan10s** [comment]
Manual entry by GTID won't include the access card number, sounds like that's fine though?

**Comment by kberzinch** [comment]
Yes, that's fine, a GTID is still the primary identifier for attendance.
